dev-util/intel-graphics-compiler/intel-graphics-compiler-1.0.3445.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_ECLASS=cmake
+inherit cmake-multilib llvm
+
+DESCRIPTION="LLVM-based OpenCL compiler targetting Intel Gen graphics hardware"
+HOMEPAGE="https://github.com/intel/intel-graphics-compiler"
+SRC_URI="https://github.com/intel/${PN}/archive/igc-${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+
+COMMON="sys-devel/llvm:=[${MULTILIB_USEDEP}]
+	dev-libs/opencl-clang:=[${MULTILIB_USEDEP}]"
+DEPEND="${COMMON}"
+RDEPEND="${COMMON}"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-1.0.9-no_Werror.patch
+	"${FILESDIR}"/0001-Fix-for-building-separated-dynamic-llvm-libs.patch
+)
+
+S="${WORKDIR}"/${PN}-igc-${PV}
+
+multilib_src_configure() {
+	# Select the same slot as the best opencl-clang
+	OCL_CLANG_VER="$(best_version dev-libs/opencl-clang:=)"
+	einfo "Selecting $OCL_CLANG_VER"
+	LLVM_SLOT=${OCL_CLANG_VER:22:1}
```

Comment:
Pretty sure this will break once opencl-clang has been slotted for LLVM10.

Reply:
Also, this too should be a local/lowercased variable.

Reply:
renamed, made local.